build.psm1

```diff
@@ -719,9 +730,13 @@ function Start-PSPester {
 
     Write-Verbose "Running pester tests at '$path' with tag '$($Tag -join ''', ''')' and ExcludeTag '$($ExcludeTag -join ''', ''')'" -Verbose
     Publish-PSTestTools
+
```

Comment:
If we are going to require everyone to use `Start-PSPester` which btw someone misnamed this function, it should be `Invoke-PSPester` since it is synchronous, it should have the `-testname` parameter.